modular_skyrat/modules/company_imports/code/armament_datums/akh_frontier.dm

```diff
@@ -10,11 +10,11 @@
 
 /datum/armament_entry/company_import/akh_frontier/basic/omni_drill
 	item_type = /obj/item/screwdriver/omni_drill
-
+/* BUBBER REMOVAL START
 /datum/armament_entry/company_import/akh_frontier/basic/prybar
 	item_type = /obj/item/crowbar/large/doorforcer
 	restricted = TRUE
-
+BUBBER REMOVAL END*/
```

Comment:
With the nerf below, does this need to be commented out?

Reply:
I personally don't like the fact that you can get a doorjack for 100cr, even if it's one use.